simple_email_confirmation/tests/tests.py

```diff
@@ -10,11 +10,13 @@
 from ..exceptions import (
     EmailConfirmationExpired, EmailIsPrimary, EmailNotConfirmed,
 )
-from ..models import EmailAddress, get_user_primary_email
+from ..models import EmailAddress, get_user_primary_email, get_email_address_model
```

Comment:
Might be a better way to do this. Definitely import `get_email_address_model` directly from the package rather than `.models`.

```suggestion
from simple_email_confirmation import get_email_address_model
from simple_email_confirmation.models import EmailAddress, get_user_primary_email
from simple_email_confirmation.exceptions import (
    EmailConfirmationExpired, EmailIsPrimary, EmailNotConfirmed,
    EmailConfirmationExpired, EmailIsPrimary, EmailNotConfirmed,
)
from simple_email_confirmation.signals import (
    email_confirmed, unconfirmed_email_created, primary_email_changed,
    email_confirmed, unconfirmed_email_created, primary_email_changed,
)
```

Reply:
Good catch on these! Sorry I missed them. I will update.